app/controllers/scheduled_messages_controller.rb

```diff
@@ -37,6 +37,14 @@ def update
     end
   end
 
+  def destroy
+    @scheduled_message = ScheduledMessage.find(params[:id])
```

Comment:
this does not need to be an instance var since we don't reference it in the view. Can just be `scheduled_message` :)